Task_objects/index.js

```diff
@@ -1,24 +1,42 @@
-const cat = {
-  name: "Bertie",
-  breed: "Cymric",
-  color: "white",
-  greeting: function () {
-    console.log("Meow!");
-  },
-};
-
-// Write the code for 1st task below
-
-
-//Write your code above
-
-
-
-
-//Write your 2nd task below this
-
-
-
-//Write your code above
-
-
+const cat = {
+  name: "Bertie",
+  breed: "Cymric",
+  color: "white",
+  greeting: function () {
+    console.log("Meow!");
+  },
+};
+
+// Write the code for 1st task below
+let catName = cat['name'];
+cat.greeting;
+cat.color = 'Black';
+
+//Write your code above

```

Comment:
Ensure there is a comma separating the objects when logging multiple objects to the console on line 39.